### DISCUSSION TITLE
LQI in dashboard vs. map?

### DISCUSSION THREAD

**Discussion by habitoti** [Koenkk/zigbee2mqtt#9790] [General]
I have a question on what exactly the LQI display shows. In the dashboard, I have mostly very high values, like > 100, mostly even 255:

![2021-11-22_17-42-33](https://user-images.githubusercontent.com/220421/142902572-1b998a72-d426-49f3-9e5e-7cca7582d249.png)

When I do a network map, though, I see very low values partially:

![2021-11-22_17-34-19](https://user-images.githubusercontent.com/220421/142903107-8d365880-7062-4043-8ed1-52dff2d521aa.png)

I am not exactly sure how to interpret those and why they are so inconsistent. Any insights welcome.

Thanks, habitoti

**Comment by Koenkk** [comment]
Are you using a conbee II adapter? If yes this is a known bug in the firmware, Dresden electronic is working on a fix.https://github.com/Koenkk/zigbee2mqtt/issues/4630

**Comment by habitoti** [comment]
Yes, it's a Conbee. Just updated to latest firmware and noticed this very change in behaviour. So thanks, I'll track that ticket then for another update.

**Comment by dsduarte** [comment]
I have this situation here too... It happend using CC2531, using a Sonoff Bridge over TCP and I just had them swaped to a Sonoff Zigbee USB dongle. With all those 3 coordinatiors this behavior is the same.... here is the current situation using the sonoff usb dongle:

![image](https://user-images.githubusercontent.com/19639490/144428541-ee1cb291-4b8c-499e-afd0-c0cc596d1a16.png)

![image](https://user-images.githubusercontent.com/19639490/144428599-f0db916f-6285-4dc9-b61e-7d4da8d573ef.png)

On network map, there are even LQI showing "0" and others showing nothing, with no lines.
